package.json

```diff
@@ -157,7 +157,8 @@
     "permissions": [
       "theme",
       "storage",
-      "tabs"
+      "tabs",
+      "management"
```

Comment:
ahh ok, I misunderstood. I thought you suggested that I should do this here. So that will definitely simplify things i think. I had this before actually but then when i re-read the tickets/comments I changed it 🤷 

So anyway, just to clarify hitting "Revert All" will default back to FF's default theme (regardless of what FF theme is set)